docs/source/sysadmin_guide/configuring_sawtooth/poet_sgx_enclave_configuration_file.rst

```diff
@@ -49,3 +49,15 @@ The ``poet_enclave_sgx.toml`` configuration file has the following options:
   Identifies the PEM-encoded certificate file that was submitted to Intel in
   order to obtain a SPID. Default: none. Specify the full path to the
   certificate file.
+  This pem file can be created from ``cert.crt`` and ``cert.key`` files
+  with command:
+
+  .. code-block:: console 
+
+    $ cat cert.crt cert.key > cert.pem
+
+  Or, from ``cert.pfx`` file with command:
```

Comment:
Please remove the comma after "Or" and change the end to "with this command:"

Reply:
Agree with removal of the comma.  Should this also be "this command"?